src/Common/tests/System/Net/Capability.Sockets.cs

```diff
@@ -47,5 +47,10 @@ public static bool IPv4Support()
         {
             return Socket.OSSupportsIPv4;
         }
+
+        public static bool UnixDomainSocketsSupport()
+        {
+            return !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
All non-windows platforms are Unix-based, so yes, they will support Unix Domain Sockets. Please note that this is non-shipping test API. If at some point some future platform won't support domain sockets, it'll be captured by the test.
However, there are already SupportPlatform attributes on the test themselves. So this property is kind of redundant now. I've added it to be consistent with other network capabilities Support() functions; it doesn't bring in much value now. I can remove it to avoid the duplication.